cookbook/generate_ligand_network.ipynb

```diff
@@ -20,25 +20,10 @@
   },
```

Comment:
<p>It's my understanding that we want to phase out perses from tutorials.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/274/discussion/#comment-2608716167'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>